package/com.unity.formats.usd/Tests/Editor/ExportTests.cs

```diff
@@ -117,5 +116,46 @@ public void ExportMultipleSiblingGameObjectsWithSameName_AllGameObjectsHaveCorre
             }
             Assert.AreEqual(cubes.Length, exportedPrims.Count, "One or more GameObjects don't have a corresponding Prim");
         }
+
+        class CameraRelated : USDExportTests
```

Comment:
Out of curiosity, what's the benefit of this nested class/ inheritance structure over just having this lass outside of USDExportTests? I think  'ExportCameraTests' or some such might be a clearer name too :)

Reply:
I wanted to sub-categorize / sub-group the test into a more specific part within the USDExportTests, but yeah, I guess I could just have them as a separate class all together instead - @IlliaKo also mentioned something similar at a call today as well